### PR TITLE
Improve generate systemd format

### PR DESCRIPTION
**Pull request by rhatdan**
Fixes: https://github.com/containers/podman/issues/14897

Followup to #13814

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
podman systemd generate output has been improved by adding newlines on options in the unit files.
```

### CODE REVIEW

File: docs/source/markdown/podman-generate-systemd.1.md [hunk 1]
```diff
@@ -177,8 +177,11 @@ ExecStart=/usr/local/bin/podman run
 	--cgroups=no-conmon
 	-d
 	-dit alpine
-ExecStop=/usr/local/bin/podman stop --ignore --cidfile %t/%n-cid -t 10
-ExecStopPost=/usr/local/bin/podman rm --ignore -f --cidfile %t/%n-cid
+ExecStop=/usr/local/bin/podman stop --ignore
```
Comment (vrothberg) [inline docs/source/markdown/podman-generate-systemd.1.md:180]: Shouldn't the lines be escaped?

**Review by vrothberg — COMMENTED** [review]
Other than the nit LGTM
@Luap99 PTAL

File: docs/source/markdown/podman-generate-systemd.1.md (hunk 1 above)
Comment (rhatdan) [reply docs/source/markdown/podman-generate-systemd.1.md]: Yes, Fixed.

**Review by vrothberg — COMMENTED** [review]
LGTM
Restarted the flaked job.

**Review by Luap99 — COMMENTED** [review]
/lgtm
/hold

**Pull request merged**